staging/cse/windows/debug/collect-windows-logs.ps1

```diff
@@ -132,22 +132,64 @@ if ($res) {
   Write-Host "Collecting Containerd running containers - tasks"
   & ctr.exe -n k8s.io t ls > "$ENV:TEMP\$timeStamp-containerd-tasks.txt"
   $paths += "$ENV:TEMP\$timeStamp-containerd-tasks.txt"
+
+  Write-Host "Collecting Containerd running containers - snapshot"
+  & ctr.exe -n k8s.io snapshot ls > "$ENV:TEMP\$timeStamp-containerd-snapshot.txt"
+  $paths += "$ENV:TEMP\$timeStamp-containerd-snapshot.txt"
 }
 else {
   Write-Host "ctr.exe command not available"
 }
 
-# log containers the CRI plugin is aware of, and their state.
-Write-Host "Collecting CRI plugin containers"
+# log containers, pods and images the CRI plugin is aware of, and their state.
 $res = Get-Command crictl.exe -ErrorAction SilentlyContinue
 if ($res) {
+  Write-Host "Collecting CRI plugin containers"
   & crictl.exe ps -a > "$ENV:TEMP\$timeStamp-cri-containerd-containers.txt"
   $paths += "$ENV:TEMP\$timeStamp-cri-containerd-containers.txt"
+
+  Write-Host "Collecting CRI plugin pods"
+  & crictl.exe pods > "$ENV:TEMP\$timeStamp-cri-containerd-pods.txt"
+  $paths += "$ENV:TEMP\$timeStamp-cri-containerd-pods.txt"
+
+  Write-Host "Collecting CRI plugin images"
+  & crictl.exe images > "$ENV:TEMP\$timeStamp-cri-containerd-images.txt"
+  $paths += "$ENV:TEMP\$timeStamp-cri-containerd-images.txt"
 }
 else {
   Write-Host "crictl.exe command not available"
 }
 
+# use runhcs shim diagnostic tool 
+$res = Get-Command shimdiag.exe -ErrorAction SilentlyContinue
+if ($res) {
+  Write-Host "Collecting logs of runhcs shim diagnostic tool"
+  $tempShimdiagFile = Join-Path ([System.IO.Path]::GetTempPath()) ("shimdiag.txt")
+  $shimdiagList = shimdiag.exe list
+  Set-Content -Path $tempShimdiagFile -Value $shimdiagList
+  foreach ($line in $shimdiagList) {
+    $tempResult = shimdiag.exe stacks $line
+    Add-Content -Path $tempShimdiagFile -Value ""
+    Add-Content -Path $tempShimdiagFile -Value $line
+    Add-Content -Path $tempShimdiagFile -Value $tempResult
+  }
+  $paths += $tempShimdiagFile
+}
+else {
+  Write-Host "shimdiag.exe command not available"
+}
+
```

Comment:
One last command that would be helpful to collect is : 

.\hcsdiag.exe list

Could we please add this too? Otherwise the changes look good to me.